### PR TITLE
Python Wrapper: Fix pagination

### PR DESCRIPTION
**Pull request by N-o-Z**

### CODE REVIEW

File: clients/python-wrapper/lakefs/reference.py [hunk 1]
```diff
@@ -76,10 +76,12 @@ def objects(self) -> ObjectManager:
     @staticmethod
     def _get_generator(func, *args, max_amount: Optional[int] = None, **kwargs):
         has_more = True
+        after = ""
         with api_exception_handler():
             while has_more:
-                page = func(*args, **kwargs)
+                page = func(*args, after=after, **kwargs)
```
Comment (idanovo) [inline clients/python-wrapper/lakefs/reference.py]: IMO it makes more sense to update the `after` value inside of the kwargs.
This is not an urgent matter.

Comment (idanovo) [reply clients/python-wrapper/lakefs/reference.py]: Just resolved by you 🙂

**Review by idanovo — APPROVED** [review]
LGTM

**Comment by N-o-Z** [comment]
@idanovo  Thanks!

**Pull request merged**